force-app/main/default/lwc/miscNotificationModules/__tests__/miscNotificationModules.test.js

```diff
@@ -0,0 +1,180 @@
+import { createElement } from 'lwc';
+import LightningAlert from 'lightning/alert';
+import LightningConfirm from 'lightning/confirm';
+import LightningPrompt from 'lightning/prompt';
+
+import MiscNotificationModules from 'c/miscNotificationModules';
+
+describe('c-misc-notification-modules', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    // Helper function to wait until the microtask queue is empty. This is needed for promise
+    // timing when calling imperative Apex.
+    async function flushPromises() {
+        return Promise.resolve();
+    }
+
+    it('shows an alert notification', async () => {
+        // Create initial element
+        const element = createElement('c-misc-notification-modules', {
+            is: MiscNotificationModules
+        });
+        document.body.appendChild(element);
+
+        LightningAlert.open = jest.fn().mockResolvedValue();
+
+        // Select button for simulating user input
+        const alertButton = element.shadowRoot.querySelector(
+            'lightning-button.alertButton'
+        );
+        alertButton.click();
+
+        // Wait for any asynchronous DOM updates
+        await flushPromises();
+
+        // Check if event has been fired
+        expect(LightningAlert.open).toHaveBeenCalled();
+        expect(LightningAlert.open.mock.calls).toHaveLength(1);
```

Comment:
I don't think that we need to check the number of time the method has been called but if you do, you can replace the two lines by just this:
```js
expect(LightningAlert.open).toHaveBeenCalledTimes(1);
```